test_average (__main__.TestExercice) ... ok
test_square (__main__.TestExercice) ... ok
test_square_root (__main__.TestExercice) ... ok
test_to_celsius (__main__.TestExercice) ... FAIL
test_to_degrees (__main__.TestExercice) ... ERROR
test_to_farenheit (__main__.TestExercice) ... FAIL
test_to_radians (__main__.TestExercice) ... ok

======================================================================
ERROR: test_to_degrees (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 80, in test_to_degrees
    output = [exercice.to_degrees(v) for v in values]
  File "test_exercice.py", line 80, in <listcomp>
    output = [exercice.to_degrees(v) for v in values]
  File "/home/runner/work/c04-ch3-exercices-AdamEL1/c04-ch3-exercices-AdamEL1/exercice.py", line 25, in to_degrees
    full_def
NameError: name 'full_def' is not defined

======================================================================
FAIL: test_to_celsius (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 98, in test_to_celsius
    'Mauvais calcul de farenheit -> celsius'
AssertionError: Lists differ: [-17.77777777777778, -31.666666666666668, 232.7777777777778] != [-17.77777777777778, -31.666666666666664, 232.77777777777777]

First differing element 1:
-31.666666666666668
-31.666666666666664

- [-17.77777777777778, -31.666666666666668, 232.7777777777778]
?                                        ^                  ^

+ [-17.77777777777778, -31.666666666666664, 232.77777777777777]
?                                        ^                  ^^
 : Mauvais calcul de farenheit -> celsius

======================================================================
FAIL: test_to_farenheit (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 110, in test_to_farenheit
    'Mauvais calcul de celsius -> farenheit'
AssertionError: Lists differ: [32.0, -13.0, 843.8] != [32.0, -13.0, 843.8000000000001]

First differing element 2:
843.8
843.8000000000001

- [32.0, -13.0, 843.8]
+ [32.0, -13.0, 843.8000000000001]
?                    ++++++++++++
 : Mauvais calcul de celsius -> farenheit

----------------------------------------------------------------------
Ran 7 tests in 0.002s

FAILED (failures=2, errors=1)
